exporter/collector/config.go

```diff
@@ -28,23 +28,33 @@ const (
 type Config struct {
 	ProjectID string `mapstructure:"project"`
 	UserAgent string `mapstructure:"user_agent"`
-	Endpoint  string `mapstructure:"endpoint"`
```

Comment:
Just calling out, I think this is the first breaking change to trace portion from our exporter rewrite. Should we keep this for backward compat and mark it deprecated or are we ok making this?

Reply:
That is correct.  I've thought about how to handle that, but didn't write it here:

In the upstream collector, we should leave items that we removed here in the configuration.  We should still pass them to the opencensus exporter.  But when the feature gate is enabled, we should thrown an error if a user tries to set them.  Once the feature gate is GA, we can safely rip out the old exporter and old config options.